src/test/article.spec.js

```diff
@@ -213,3 +220,418 @@ describe('GET api/v1/articles', () => {
       });
   });
 });
+
+describe('POST api/v1/articles/', () => {
+  it('should throw a 400 status code when creating an article with invalid token', async () => {
+    beforeEach(async () => {
+      await db.Bookmark.destroy({ cascade: true, truncate: true });
+      await db.User.destroy({ cascade: true, truncate: true });
+      await db.Article.destroy({ cascade: true, truncate: true });
+    });
+
+    const token = 'helper.jwtSigner(newUser);';
+
+    const response = await server()
+      .post(`${ARTICLES_API}`)
+      .set('x-access-token', token)
+      .send(articleWithNoTitle);
+    expect(response.status).to.equal(400);
```

Comment:
this is an authentication error. Change it to 401 and update auth middleware accordingly.